main.cpp

```diff
@@ -409,6 +437,39 @@ int main(int argc, char *argv[])
     opt.weather_coefficient_file->required = NO;
     opt.weather_coefficient_file->guisection = _("Weather");
 
+    // Survival rate
+
+    opt.survival_rate_month = G_define_option();
+    opt.survival_rate_month->type = TYPE_INTEGER;
+    opt.survival_rate_month->key = "survival_month";
+    opt.survival_rate_month->label =
+        _("Month when the pest or patogen dies due to low temperature");
```

Comment:
Inconsistent description with survival day. Not sure if it always has to be due to low temperature.

Reply:
Sorry, more a copy-paste issue. Made it general like the day description.